plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1890,6 +1890,53 @@ private static CommitInfoEntry getCommitInfoEntry(
                 Optional.of(true));
     }
 
+    @Override
+    public void setTableProperties(ConnectorSession session, ConnectorTableHandle tableHandle, Map<String, Optional<Object>> properties)
```

Comment:
Handling for unsupported set table properties is missing. Please fix and add a test case.

Reply:
Unsupported properties value will already be handled in

https://github.com/trinodb/trino/blob/ce40c8a6fbe2198e01368c9f8468f5da102fa086/core/trino-main/src/main/java/io/trino/execution/SetPropertiesTask.java#L80-L88

Because I am adding validation in `DeltaLakeTableProperties`
https://github.com/trinodb/trino/blob/ce40c8a6fbe2198e01368c9f8468f5da102fa086/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java#L84-L95

Here is the test:
https://github.com/trinodb/trino/blob/ce40c8a6fbe2198e01368c9f8468f5da102fa086/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java#L1006

Reply:
Try changing `checkpoint_interval` table property.

Reply:
Sorry, I misunderstood your comment. Addressed it.